src/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/2df075da03.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="index.js" type="module"></script>
+    <title>Pokemon Cards</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo"></div>
+      <p class="pokeCounter"></p>
+      <a href="#">Berries</a>
+      <a href="#">Locations</a>
+    </header>
+    <section class="container">
+      <div id="cards-container"></div>
+    </section>
+    <section id="popup">
+     <!--  <span id="close">X</span>
+      <div id="container">
+        <img src="#" alt="pokemon" id="pokemonImg">
+        <h2 id="pokemonName"></h2>
+        <div id="pokeData">
+          <div class="column">
+            <p id="Weight">Weight:</p>
+            <p>Types:</p>
+            <ol id="types"></ol>
+          </div>
+          <div class="column">
+             <p id="Height">Height:</p>
+             <p id="Species">Species:</p>
+          </div>
+        </div>
+        <div id="commentsAndCount">
+          <h3 id="commentsTitle">Comments:</h3><span id="commentsCount"></span>
+        </div>
+        <div><ul id="comments"></ul></div>
+        <div id="commentContainer">
+          <h4>Add a comment</h4>
+          <input type="text" placeholder="Your name" id="userName">
+          <textarea name="comment" id="comment" placeholder="Your insights"></textarea>
+          <input type="button" value="Comment" id="commentBtn">     
+        </div>
+      </div> -->
```

Comment:
It would be better if we clean up our code base from time to time, to keep it shiny and readable :wink:

Kindly remove any commented code that doesn't help your reader to understand your code :ballot_box_with_check: